www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -255,64 +246,70 @@ const Filter = (): JSX.Element => {
         </div>
       }
       expandableFilters={
-        <div className={clsx([classes.grid, classes.criteriaRow])}>
-          <Typography className={classes.filterLineLabel} variant="subtitle1">
-            {t(labelCriterias)}
-          </Typography>
-          <div />
-          <MultiAutocompleteField
-            options={availableResourceTypes}
-            label={t(labelResource)}
-            onChange={changeResourceTypes}
-            value={resourceTypes || []}
-            openText={`${t(labelOpen)} ${t(labelResource)}`}
-            limitTags={2}
-            fullWidth
-          />
-          <MultiAutocompleteField
-            options={availableStates}
-            label={t(labelState)}
-            onChange={changeStates}
-            value={states || []}
-            openText={`${t(labelOpen)} ${t(labelState)}`}
-            limitTags={1}
-            fullWidth
-          />
-          <MultiAutocompleteField
-            options={availableStatuses}
-            label={t(labelStatus)}
-            onChange={changeStatuses}
-            value={statuses || []}
-            openText={`${t(labelOpen)} ${t(labelStatus)}`}
-            fullWidth
-            limitTags={2}
-          />
-          <MultiConnectedAutocompleteField
-            getEndpoint={getConnectedAutocompleteEndpoint(
-              buildHostGroupsEndpoint,
-            )}
-            label={t(labelHostGroup)}
-            onChange={changeHostGroups}
-            value={hostGroups || []}
-            openText={`${t(labelOpen)} ${t(labelHostGroup)}`}
-            field="name"
-            fullWidth
-          />
-          <MultiConnectedAutocompleteField
-            getEndpoint={getConnectedAutocompleteEndpoint(
-              buildServiceGroupsEndpoint,
-            )}
-            label={t(labelServiceGroup)}
-            onChange={changeServiceGroups}
-            value={serviceGroups || []}
-            openText={`${t(labelOpen)} ${t(labelServiceGroup)}`}
-            field="name"
-            fullWidth
-          />
-          <Button color="primary" onClick={clearAllFilters}>
-            {t(labelClearAll)}
-          </Button>
-        </div>
+        <ParentSize>
+          {({ width }): JSX.Element => {
+            const limitTags = width < 1000 ? 1 : 2;
```

Comment:
We'll may need to improve this calculation, to add more than 2 tags when possible

Here, I've selected `Down`, `Up`, and `Ok`. All these tags should be displayed
![image](https://user-images.githubusercontent.com/11978823/105465951-e3feb300-5c93-11eb-8b51-fb368cbc4516.png)

Reply:
Yes, this can be improve. The thing we don't want though is having wrapped fields in case all filters are filled up